google-cloud-debugger/google-cloud-debugger.gemspec

```diff
@@ -0,0 +1,39 @@
+# -*- encoding: utf-8 -*-
+require File.expand_path("../lib/google/cloud/debugger/version", __FILE__)
+
+Gem::Specification.new do |gem|
+  gem.name          = "google-cloud-debugger"
+  gem.version       = Google::Cloud::Debugger::VERSION
+
+  gem.authors       = ["Heng Xiong"]
+  gem.email         = ["hexiong@google.com"]
+  gem.description   = "google-cloud-debugger is the official library for Stackdriver Debugger."
+  gem.summary       = "API Client and instrumentation library for Stackdriver Debugger"
+  gem.homepage      = "http://googlecloudplatform.github.io/google-cloud-ruby/"
+  gem.license       = "Apache-2.0"
+
+  gem.files         = `git ls-files -- lib/* ext/*`.split("\n") +
+                      ["README.md", "LICENSE", ".yardopts"]
+  gem.require_paths = ["lib"]
+
+  gem.required_ruby_version = ">= 2.3.0"
```

Comment:
Is this necessary? The other Google Cloud gems are supporting ruby 2.0+. If this gem is added as a dependency to the `google-cloud` gem it would block its use on all rubies < 2.3.